events/story_cycles/wc_story_cycle_horde_invasion_events.txt

```diff
@@ -47,39 +47,109 @@ WCHOI.2 = {
 	}
 	right_portrait = scope:stormwind_holder
 
-	option = { # Corrupted orcs option
-		trigger = {
-			NOT = { 
-				faith = faith:orcish_shamanism
+	option = {
+		name = {
+			text = {
+				first_valid = {
+					triggered_desc = { # Corrupted orcs option
+						trigger = { faith = faith:orcish_fel }
+						desc = WCHOI_2_OPT_A
+					}
+					triggered_desc = { # Frostwolf option
+						trigger = { faith = faith:orcish_shamanism }
+						desc = WCHOI_2_OPT_B
+					}
+					desc = WCHOI_2_OPT_C # Outsider option
+				}
 			}
-			faith = faith:orcish_fel
 		}
-		name = WCHOI_2_OPT_A
 	}
+}
 
-	option = { # Frostwolf option
+# Ends the Horde Invasion when enough lands taken
+WCHOI.3 = { # Save the Horde?
+	type = character_event
+	title = WCHOI_2_TITLE
+	desc = WCHOI_3_DESC
+	theme = war
+	override_background = {
+		reference = wc_background_dark_portal
+	}
+	left_portrait = root
+
+	option = { # The orcs need their warchief!
```

Comment:
Explain what is difference here in a tooltip.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/118501678-097b6500-b73a-11eb-93d3-c316f53026e5.png)
![image](https://user-images.githubusercontent.com/46384992/118501717-139d6380-b73a-11eb-91cf-834c381ceac4.png)

</details>